runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpState.java

```diff
@@ -55,7 +55,8 @@ void reset()
 
 	/**
 	 * Resets a single skill
-	 * @param skill Skill to reset
+	 *
```

Comment:
All of these added lines need to be removed too. It's pointless whitespace and I counted 8 of them. Six here and two in XpTrackerPlugin